Documentation/freebsd-instructions.md

```diff
@@ -24,13 +24,39 @@ Install the following packages for the toolchain:
 - libunwind
 - gettext
 
-Note: LLDB is not yet available in the pkg repository on FreeBSD. The LLDB plugin is currently not being build on FreeBSD. Instructions for this will be added later.
-
 To install the packages you need:
 
-```janhenke@freebsd-frankfurt:~ % sudo pkg install bash cmake clang35 libunwind gettext```
+```
+janhenke@freebsd-frankfurt:~ sudo pkg install bash cmake clang35 libunwind gettext
```

Comment:
Undo this change to make the block consistent with the rest of the doc? (The single line and multiline versions look differently in the rendered view.)